noirup

```diff
@@ -28,6 +29,8 @@ main() {
     esac; shift
   done
 
+  FEATURES=${NOIRUP_FEATURES:-"default"}
```

Comment:
Can you add a warning that features are ignored if we end up downloading a prebuilt binary?